unit-tests/unit-tests-common.h

```diff
@@ -858,6 +858,81 @@ inline rs2::depth_sensor restart_first_device_and_return_depth_sensor(const rs2:
     return depth_sensor;
 }
 
+inline rs2::device return_first_device_of_requeired_product( const rs2::context & ctx, int product )
+{
+    rs2::device_list devices_list = ctx.query_devices( product );
+    size_t device_count = devices_list.size();
+    REQUIRE( device_count > 0 );
+    auto dev = devices_list[0];
+    return dev;
+}
+
+inline rs2::depth_sensor get_depth_sensor( rs2::device dev )
```

Comment:
Why is it needed? You can just `dev.first< rs2::depth_sensor >()`, no?

Reply:
we want to do this test only on l515 ,no?

Reply:
I don't understand the answer :) why do we need this function (it's now in `test.h`)

Reply:
done